arrow-core-data/src/main/kotlin/arrow/typeclasses/Functor.kt

```diff
@@ -130,7 +132,8 @@ interface Functor<F> : Invariant<F> {
    * }
    * ```
    */
-  fun <A> Kind<F, A>.unit(): Kind<F, Unit> = map { Unit }
+  fun <A> Kind<F, A>.unit(): Kind<F, Unit> =
```

Comment:
Should be changed in favor of `void`, now that we're here :D

Reply:
Classic "poyaque" 😏

Reply:
Btw should we deprecate this first and add a new one `void` or just go with a rename?

Reply:
I was about to ask the same :D cc @pakoito @nomisRev

Reply:
We should probably deprecate and add `void`. We'll go through a short deprecation cycle since we're trying to release `0.11.0` asap anyway.